android/src/main/res/values/strings.xml

```diff
@@ -1,18 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">Happy News</string>
-    <string name="title_activity_browser">BrowserActivity</string>
-    <string name="api_url">https://happynews-api.svendubbeld.nl</string>
+    <string name="api_url" translatable="false">https://happynews-api.svendubbeld.nl</string>
     <string name="notification_text">We\'ve got some positivity for you!</string>
     <string name="title_settings">Settings</string>
     <string name="title_settings_about">About</string>
     <string name="title_settings_language">Language</string>
     <string name="title_settings_notification">Notifications</string>
     <string name="title_settings_sources">Sources</string>
-    <string name="preference_file_key">HappyNews_Local_Settings</string>
-    <string name="preference_twitter_enabled">twitter_enabled</string>
-    <string name="preference_quotes_enabled">quotes_enabled</string>
-    <string name="preference_articles_enabled">articles_enabled</string>
-    <string name="preference_notifications">notifications</string>
-    <string name="preference_sources">notifications</string>
+    <string name="title_settings_rate_app">Rate App</string>
+    <string name="preference_file_key" translatable="false">HappyNews_Local_Settings</string>
+    <string name="preference_twitter_enabled" translatable="false">twitter_enabled</string>
+    <string name="preference_quotes_enabled" translatable="false">quotes_enabled</string>
+    <string name="preference_articles_enabled" translatable="false">articles_enabled</string>
+    <string name="preference_notifications" translatable="false">notifications</string>
+    <string name="preference_sources" translatable="false">notifications</string>
+    <string name="about_text">With Happy News you get the positivity you need to brighten up the day. We have an extensive amount of sources which we use to bring only the most positive news to your screen. No need to log in, completely free and 24/7 available, We just want to bring positivity to your day. \n\n Happy News features:\n - Constant stream of positive articles\n - Happy tweets based on carefully picked hashtags\n - Inspirational quotes to put some spring in your step\n\n At Happy News we aspire to bring positivity around the world. We want to make a hard days work more enjoyable by bringing some much needed positivity when needed.</string>
```

Comment:
Fixed.

@DaanTul We should probably fix this in the Play Store listing as well.